src/roswire/ros2/state.py

```diff
@@ -28,31 +28,61 @@ class ROS2SystemState(SystemState):
         A mapping from topics to the names of subscribers to that topic.
     services: Mapping[str, Collection[str]]
         A mapping from services to the names of providers of that service.
+    clients: Mapping[str, Collection[str]]
```

Comment:
Can you call it `service_clients` to be consistent with `action_clients`?

Reply:
Done

Reply:
It looks like the docstring is still using the old name here